apps/studio/src/features/editing-experience/components/form-builder/renderers/layouts/JsonFormsVerticalLayout.tsx

```diff
@@ -26,8 +26,9 @@ export function JsonFormsVerticalLayoutRenderer({
 
   return (
     <VStack spacing={2}>
-      {elements.map((element) => (
-        <Box key={path} w="100%">
+      {elements.map((element, index) => (
+        // eslint-disable-next-line react/no-array-index-key
+        <Box key={`${path}-${index}`} w="100%">
```

Comment:
Is adding the index due to everything within a nested object sharing the same path? and adding the index isn't a problem because we aren't changing the array's ordering or contents

Reply:
Yeah mainly because we are not going to be re-ordering the form fields, since it is constant as defined by our Isomer Schema.

Reply:
Here the path refers to the path to the element in the schema e.g. `"#/components/native/paragraph"`?

Reply:
Can't we just use one of the `element` properties to craft the key?

Reply:
Hmm no not exactly, it is the path within the context of the parent control element. For instance, when rendering the Hero component, the path is just an empty string `""`, because we are rendering within the context of that specific component. When there are nested objects (like the dropdown within the hero component), then the path would be `dropdown`, relative to the parent.

Example when I print out the path for the hero component just above line 29:

<img width="495" alt="Screenshot 2024-07-02 at 15 26 10" src="https://github.com/opengovsg/isomer/assets/27919917/63bbe8be-ec6c-44cc-813d-d21c9fcdff5a">